test/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnectorSpec.scala

```diff
@@ -49,6 +54,31 @@ class DesConnectorSpec extends NationalInsuranceRecordUnitSpec with MockitoSugar
   val mockDesNIRecord: CachingMongoService[DesNIRecordCache, DesNIRecord] = mock[CachingMongoService[DesNIRecordCache, DesNIRecord]]
   when(mockNIRecordRepo()).thenReturn(mockDesNIRecord)
 
+  val mockHttpClient: HttpClient = mock[HttpClient]
+
+  val mockMetrics: MetricsService = mock[MetricsService]
+  val mockTimerContext: Timer.Context = mock[Timer.Context]
+
+
+  override def fakeApplication(): Application = GuiceApplicationBuilder()
+    .configure(
+      "microservice.services.des-hod.port" -> server.port()
+    )
+    .overrides(
+      bind[HttpClient].toInstance(mockHttpClient),
+      bind[CachingMongoService[DesSummaryCache, DesSummary]].toInstance(mockDesSummary),
+      bind[MetricsService].toInstance(mockMetrics),
+      bind[Timer.Context].toInstance(mockTimerContext),
+      bind[DesNIRecordRepository].toInstance(mockNIRecordRepo),
+      bind[DesSummaryRepository].toInstance(mockSummaryRepo),
+      bind[DesLiabilitiesRepository].toInstance(mockLiabilitiesRepo)
+
+
+    )
+    .build()
+
+  lazy val connector: DesConnector = app.injector.instanceOf[DesConnector]
```

Comment:
Suggestion

This may not need to be lazy

Reply:
Needs to be lazy or all hell brakes loose

Reply:
ah, yes. Wiremock is in there too. You are right.